### PR TITLE
Formatting and links

### PR DESCRIPTION
**Pull request by amyconroy**
formatting updates

### CODE REVIEW

File: Portfolio/SystemImplementation.md [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
 
-In order to keep it clear which user stories were implemented during each sprint, we created a Gantt chart to be used between each of the teams working on the different subsystems. This was used in conjunction with our project's [Kanban Board](https://github.com/Team-Jag/Do-or-die-fitness-tracker/projects/1). Both of these features were useful to keep our work user focused, while ensuring cohesion between each subsystem. 
+As one of the key foundations of Agile software development is to have self-organising teams, and we were confident in the capabilities to successfuly maintain a minimum viable product during each weeks sprints, we knew that it was important to collectively focus on the most important user stories to be implemented during each sprint. In order to keep this clear, we created a Gantt chart to be used between each of the teams working on the different subsystems. This was used in conjunction with our project's [Kanban Board](https://github.com/Team-Jag/Do-or-die-fitness-tracker/projects/1). Both of these features were useful to keep our work user focused, while ensuring cohesion between each subsystem.
```
Comment (gracielawp) [inline Portfolio/SystemImplementation.md]: the first sentence is quite long, i might break it down into two sentences if that's ok

**Review by gracielawp — APPROVED** [review]
i like it bean, but i added one small review about a sentence being too long - i can reformat it if that's ok

File: Portfolio/SystemImplementation.md (hunk 1 above)
Comment (amyconroy) [reply Portfolio/SystemImplementation.md]: @gracielawp if you are a bored bean you can but i can also if you dont wanna

**Comment by amyconroy** [comment]
> i like it bean, but i added one small review about a sentence being too long - i can reformat it if that's ok

you the best bean

**Pull request merged**